api/relations/v1/relation_tuples.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+
+package api.relations.v1;
+
+import "google/api/annotations.proto";
+import "relations/v1/common.proto";
+
+option go_package = "ciam-rebac/api/relations/v1;v1";
+option java_multiple_files = true;
+option java_package = "api.relations.v1";
+
+service RelationTupleService {
+	rpc CreateTuples (CreateTuplesRequest) returns (CreateTuplesResponse) {
```

Comment:
Any argument for just `Create`, `Read` and `Delete` (without `Tuples` suffix)? (Happy to go with the style guides, etc.)

Reply:
Should we rename the messages also?

- Style guides say so
- But arguably we may want to disambiguate with the message types (they are not "housed" in the Service, i.e. not already so contextualized)